src/TwoBucks/ReportToText.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * Create Organized Report Feature US.5
+ *
+ * The ReportToText class is used when the user chooses to
+ * save the report created in the CreateProfile class in a txt file.
+ *
+ * @author Frino Jais
+ * @version 1.0
+ * @since 04-01-2020
+ */
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.PrintWriter;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class ReportToText {
+
+
+    /**
+     * This method will output the user report generated in the CreateReport class to a txt file.
+     * This txt file will also include a time stamp so the user knows when the report is from
+     * if they have multiple reports saved.
+     * @param currentUser
+     */
+    void printInfoToFile(User currentUser) {
+
+        DateTimeFormatter dateFormatter = DateTimeFormatter.ofPattern("MM/dd/yyyy HH:mm:ss");
+        LocalDateTime now = LocalDateTime.now();
+
+        File file = new File("TwoBucks User Report.txt"); //reading data from this file
```

Comment:
File name will always be the same, so the reports will always be saved over. Consider using a String to name the file with the user's name and date in order to prevent this.

Reply:
Working on this

Reply:
Done